busybox-static/tests/test.bats

```diff
@@ -0,0 +1,48 @@
+TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
+
+@test "busybox run successfully" {
+  run hab pkg exec "${TEST_PKG_IDENT}" busybox
+  [ $status -eq 0 ]
+}
+
+@test "busybox is statically linked" {
+  file /hab/pkgs/"$TEST_PKG_IDENT"/bin/busybox |grep -q statically
+  [ $? -eq 0 ]
+}
+
+@test "busybox linked applets correctly" {
+  # In the past we've seen issues where `busybox --list` would fail silently
+  # and the applet commands would not be linked, but the package would still
+  # be generated. This checks the critical commands used in the studio
+  # to ensure they're present. We don't check all though, there are >380!
+  #
+  # This is a somewhat awkward test, in that we're largely testing that our plan
+  # is correct and plan-build behaved correctly. Due to the potential for silent
+  # failures and the critical nature of this package and these commands, it's
+  # worth validating.
+  [ -s /hab/pkgs/$TEST_PKG_IDENT/bin/awk ]
```

Comment:
I know you cannot define BATS `@test`s in a loop, but maybe within a `@test` it would be possible to declare an array with all these applets then loop on them and `[ -s "/hab/pkgs/$TEST_PKG_IDENT/bin/awk$i" ]` each one?